src/project/settingup_forms.md

```diff
@@ -309,4 +309,53 @@ To use the drop-down widget in the form from Input:
+
+Another example is when you try to capture multiple photos for a single feature.
+
+In the sections below, you can see how this type of relations can be set up in QGIS and utilised in the Input app.
+
+### Project preparation in QGIS
+
+The example projects can be found:
+- [Assigning multiple inspection to a single feature](<MerginMapsProject id="documentation/forms_one-to-many-relations" />)
+- [Adding multiple photos to a single feature](<MerginMapsProject id="documentation/forms_multiple_photos" />)
+
+To configure 1-N relations in QGIS:
+- From the main menu, select **Projects** > **Properties ...**
+- In the new window, select the **Relations** tab
+- Select **Add Relation** to create a new one
+- A new window will appear, where we can define the parent and child layers and the fields to link the two layers:
+  - For **Name** type **Inspection**
+  - For the **Referenced(parent)** layer, select **manhole_locations**
+  - For **Field 1** of the **Referenced(parent)** layer, select **Manhole**
+  - For the **Referencing(child)** layer, select **inspections**
+  - For **Field 1** of the **Referencing(child)** layer, select **Manhole ID**
+
+![1-N relations in QGIS](./input_forms_many-relations1.png)
+
+This should now allow you to add multiple inspections for each manhole location (**manhole_locations** point layer). The inspections records will be stored in the **inspections** table.
+
+When you open the form for an existing record in the  **manhole_locations** point layer, you should be able to see the existing inspection records and optionally add, delete or edit the records:
+
+![1-N relations in QGIS - form view](./input_forms_many-relations1.png)
+
+The same project when you open add inspection to a manhole in Input will look like the image below:
+
+![Many photos to a single feature](./input_forms_one-to-many.png)
+
+Another use-case for having multiple photos linked to a single feature is using 1-N relation.
+
+To set up a project in QGIS, similar to the previous example, we need a unique field to link the following two tables:
+- Survey layer (containing spatial information)
+- A non-spatial table containing path to the photos
+
+The key part when linking the above to table is to avoid using **FID** field in the GeoPackage. Mergin uses the FID to consolidate changes and therefore can change the FIDs. This will result in having photos linked to the incorrect feature on the map.
+
+For that, we can create a new field and use the **uuid()** as the default value. To learn more about how this can be configured, you can see [this example](<MerginMapsProject id="documentation/forms_multiple_photos" />).
+
```

Comment:
It should be explained also how you get the image viewer in Input - that it is automatically detected that the linked layer has a field with attachment.